### PR TITLE
Remove some deprecate warnings

### PR DESCRIPTION
**Pull request by vojtechtrefny**

### CODE REVIEW

File: tools/udisksctl.c [hunk 1]
```diff
@@ -2601,20 +2601,15 @@ handle_command_dump (gint        *argc,
...
-  GTimeVal now;
-  time_t now_time;
-  struct tm *now_tm;
-  gchar time_buf[128];
-
-  g_get_current_time (&now);
-  now_time = (time_t) now.tv_sec;
-  now_tm = localtime (&now_time);
-  strftime (time_buf, sizeof time_buf, "%H:%M:%S", now_tm);
+  GDateTime *now = g_date_time_new_now_local ();
```
Comment (tbzatek) [inline tools/udisksctl.c]: Need to call `g_date_time_unref()` once done with it?

Comment (vojtechtrefny) [reply tools/udisksctl.c]: Oops, fixed.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Pull request merged**